src/main/webapp/modal-tester.html

```diff
@@ -7,11 +7,11 @@
     <script src="modal-activator.js"></script>
   </head>
   <body>
-    <Button> Load Results </Button>
+    <Button data-modal-target="#modal"> Load Results </Button>
     <div class="modal" id="modal" name="results-popup">
       <div class="modal-header"> 
         <div class="modal-title"> Results </div>
-        <button class="exit-button"> &times;</button>
+        <button data-modal-close-button class="exit-button"> &times;</button>
```

Comment:
not sure how this works, but seems incomplete, does this not need to = anything?

Reply:
Correct. It's an attribute without a corresponding value. It's useful for collecting it and like-objects in JS.